bolt04/onion-error-test.json

```diff
@@ -0,0 +1,43 @@
+{
+  "comment": "A simple error returned by node hops[4], the public/private keys and shared_secrets are identical to the ones used in `onion-test-v0.json`",
```

Comment:
Yay JSON!

Reply:
I know :wink: But it was the simplest way I could figure out to add a JSON-compatible comment.